pom.xml

```diff
@@ -317,6 +317,23 @@
           </dependency>
         </dependencies>
       </plugin>
+      <plugin>
+        <groupId>org.gaul</groupId>
+        <artifactId>modernizer-maven-plugin</artifactId>
```

Comment:
I am not sure we want this plugin and the dependency.
I see it is an useful tool.

Maybe we should start a thread on dev@maven.apache.org

@rfscholte

Reply:
@gaul 
hehe, a good trick ;-)
I will copy this config for myself, but you should really remove it here.

Reply:
Done.